presto-kafka/src/main/java/io/prestosql/plugin/kafka/encoder/avro/AvroRowEncoder.java

```diff
@@ -59,6 +59,7 @@ public AvroRowEncoder(ConnectorSession session, List<EncoderColumnHandle> column
         for (EncoderColumnHandle columnHandle : this.columnHandles) {
             checkArgument(columnHandle.getFormatHint() == null, "Unexpected format hint '%s' defined for column '%s'", columnHandle.getFormatHint(), columnHandle.getName());
             checkArgument(columnHandle.getDataFormat() == null, "Unexpected data format '%s' defined for column '%s'", columnHandle.getDataFormat(), columnHandle.getName());
+            checkArgument(columnHandle.getMapping() == null, "Mapping not provided for column '%s'", columnHandle.getName());
```

Comment:
Docs will need updating to point out that mapping is mandatory for INSERTs to work.